README.md

```diff
@@ -1,8 +1,24 @@
 # Booster Catalog
-Set of known example applications (Boosters) conforming to the minimal set of requirements necessary to be served by launch.openshift.io.
-
-- next: this is the branch from which we'll cut the next tag. Production does not use this branch, it is only used in the staging environment.
-- openshift-online-free; this branch is used in staging AND production. Only the Boosters capable of running in the OSO Free environment are included here.
+Set of known example applications (Boosters) conforming to the minimal set of requirements necessary to be served by developers.redhat.com/launch.
+
+Branching and tagging strategy:
+- `master`:
+    - should always point to the `master` branches of the boosters
+        - if a particular variant of a booster isn't available in a branch, then it should point to the latest tag
+    - can be updated at any time, because it's not used in any Fabric8 Launcher instance we run
+- `next`:
+    - should always point to the latest tags of the boosters
+    - can be updated at any time, because it's only used in the staging Fabric8 Launcher instance we run
+    - booster catalog releases are cut from this branch
+        - the release is only cut when all the boosters dependencies are available publicly
+        - the catalog release is used in the production Fabric8 Launcher instance we run, but only when you target an OpenShift Online Pro cluster
+        - the catalog release is also used when you install Fabric8 Launcher locally from the template
+- `openshift-online-free`:
```

Comment:
My opinion is that this branch should not exist, rather a single file should be used to register which boosters are available on OpenShift Online free version and then the info should be derived from latest catalog release. As it is, it's just redundant and a maintenance liability.

Reply:
Yea makes sense; probably file a feature request at https://github.com/fabric8-launcher/launcher-planning

Reply:
@metacosm I think the issue is that boosters could be specifically adjusted to run on the free version. So perhaps instead of branches we should be using some other system to determine which catalog is used in which environment.

A suggestion might be to add a special tag to the .yaml file named `environment:`. If it doesn't exist the booster appears in all environments (dev, staging, production). If it _does_ exist you can give it a list of environments where it should appear.  (We could even do something more fancy and allow you to say `environment: -free` meaning that you want it to appear everywhere _except_ in the free version)